### PR TITLE
Dockerfile

### PR DESCRIPTION
**Pull request by adrianherrera**
Also added output directory option to cclyzer script, which makes it easier to use with mounted volumes in a Docker container.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ the system's package manager should suffice.
 
 #### Ubuntu 20.04
 
-    apt install llvm
+    apt install llvm libclang-dev
```
Comment (ktrianta) [inline README.md:32]: I created a separate commit for this. Can you please delete this change?

File: cclyzer.sh [hunk 2]
```diff
@@ -42,11 +49,16 @@ if [ ! -f "$INPUT_FILE" ]; then
     exit 1
 fi
 
-rm -rf ./facts/
-mkdir facts
-bin/fact-generator -o facts/ "$INPUT_FILE"
-python3 cti.py facts/
-mkdir -p results
+if [ ! -d "$OUTPUT_DIR" ]; then
+    echo Output direcctory "$OUTPUT_DIR" not found!
```
Comment (ktrianta) [inline cclyzer.sh:53]:
```suggestion
    echo Output directory "$OUTPUT_DIR" not found!
```

File: README.md (hunk 1 above)
Comment (ktrianta) [reply README.md:32]:
```suggestion
    apt install llvm
```

**Review by ktrianta — CHANGES_REQUESTED** [review]
Thank you a lot for your pull request! I have created two small suggestions that should be addressed before merging.

**Comment by adrianherrera** [comment]
Thanks @ktrianta! Changes have been made.

**Comment by ktrianta** [comment]
Thank you!

**Pull request merged**